packages/react/src/components/Accordion/AccordionItem.tsx

```diff
@@ -82,12 +81,6 @@ interface AccordionItemProps {
    * The accordion title.
    */
   title?: ReactNode;
-
-  /**
-   * The callback function to run on the `onAnimationEnd`
-   * event for the list item.
-   */
-  handleAnimationEnd?: AnimationEventHandler<HTMLLIElement>;
```

Comment:
Removing a prop would constitute a breaking change, which we can't do right now. Is there a path to reinstating this prop to be called when the animation ends?

Reply:
Got it! I have resolved this issue by re-adding the event.